From bb5d738786bddffa2d49788ca79c3bb7f59df22f Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:33:14 -0400
Subject: [alsa] hda: Fix onboard audio on Intel H97/Z97 chipsets

Message-id: <140972823573-309-patchset-jkysela@redhat.com>
Patchwork-id: 91005
O-Subject: [RHEL 7.1 PATCH 308/442] ALSA: hda - Fix onboard audio on Intel H97/Z97 chipsets
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The recent Intel H97/Z97 chipsets need the similar setups like other
Intel chipsets for snooping, etc.  Especially without snooping, the
audio playback stutters or gets corrupted.  This fix patch just adds
the corresponding PCI ID entry with the proper flags.

Reported-and-tested-by: Arthur Borsboom <arthurborsboom@gmail.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 77f07800cb456bed6e5c345e6e4e83e8eda62437)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
index 65c2b37..97b4c4e 100644
--- a/sound/pci/hda/hda_intel.c
+++ b/sound/pci/hda/hda_intel.c
@@ -1743,6 +1743,9 @@ static const struct pci_device_id azx_ids[] = {
  /* Lynx Point */
  { PCI_DEVICE(0x8086, 0x8c20),
    .driver_data = AZX_DRIVER_PCH | AZX_DCAPS_INTEL_PCH },
+ /* 9 Series */
+ { PCI_DEVICE(0x8086, 0x8ca0),
+   .driver_data = AZX_DRIVER_PCH | AZX_DCAPS_INTEL_PCH },
  /* Wellsburg */
  { PCI_DEVICE(0x8086, 0x8d20),
    .driver_data = AZX_DRIVER_PCH | AZX_DCAPS_INTEL_PCH },
-- 
1.7.1